src/functions-and-arrays.js

```diff
@@ -1,41 +1,103 @@
-// Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
-
+// Iteration #1: Find the maximum **********DONE************
+function maxOfTwoNumbers(a, b) {
+  if (a > b) {
+    return a;
+  } else {
+    return b;
+  }
+}
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+let longWord = 0;
+let longestWord = '';
+
+function findLongestWord(words) {
+  if (words.length == 0) {
+    return null;
+  } else {
+    for (let i = 0; i < words.length; i++) {
+      if (words[i].length > longWord) {
+        longWord = words[i].length;
+        longestWord = words[i];
```

Comment:
IMPORTANTE

Las variables que necesites en una función declaralas dentro de la función. De esa forma se resetearan cada vez que la ejecutes. Si no después de lanzar la función por primera vez la variable longestWord ya contendrá alguna palabra.

```suggestion

function findLongestWord(words) {
  let longWord = 0;
  let longestWord = '';

  if (words.length == 0) {
    return null;
  } else {
    for (let i = 0; i < words.length; i++) {
      if (words[i].length > longWord) {
        longWord = words[i].length;
        longestWord = words[i];
```